products/content/tier-2-content-IA-and-design/cope-cms-faq-library/engineering/README.md

```diff
@@ -1,5 +1,33 @@
 # Exposing Q&A on New Content Types
 
+## CMS Instructions: Reusable Q&A Implementation Runbook
+For existing content types, edit the field_content_block field (Main content) if it exists on the content type, select to add the Q&A paragraph type, then save.
+
+Now when editors want to add an existing reusable Q&A they will see it as one of the options for adding main content.
+
+1. In the Drupal UI > Structure > Content types and select your content type.
```

Comment:
SHOULD:

> select your content type.

'Manage fields' is the link that should be clicked. There is no link for a content type on this page.

Reply:
Changed to "In the Drupal UI > Structure > Content types, click "Manage fields" for your content type."